Tools/MonoGame.Effect.Compiler/Effect/ShaderProfile.Vulkan.cs

```diff
@@ -222,7 +223,53 @@ internal override ShaderData CreateShader(ShaderResult shaderResult, string shad
                 }
                 toolArgs += "\"" + hlslFile + "\"";
 
-                toolResult = ExternalTool.Run("dxc", toolArgs, out stdout, out stderr);
+                var processInfo = new ProcessStartInfo
```

Comment:
For sure move this all into a shared helper method so it can be used from other shader profiles as needed.

Reply:
This piece of code is temporary, I have something much better in mind for it but thats for a separate PR.

> This breaks mgfxc invoking dxc, I plan on making a bounty for getting a MonoGame.Tool.Dxc after fixing up some stuff with how we handle dotnet tooling stuff.